docs: Add stale.yml file for stalebot

ekowtaylor opened the pull request:
Signed-off-by: Ekow Taylor <ekowtaylor@gmail.com>

## Summary
This implements the config file for a GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.

## Test Plan
Privately on another repo

## Additional Information

ekowtaylor committed:
Add stale.yml file for stalebot

Signed-off-by: Ekow Taylor <ekowtaylor@gmail.com>

This implements the config file for a GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.

ekowtaylor committed:
docs: Add stale.yml file for stalebot 

Signed-off-by: Ekow Taylor <ekowtaylor@gmail.com>

This implements the config file (now with additional changes) for a GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.

ekowtaylor commented:
@electronjoe , @themarwhal please stamp this when you have sometime. The app has already been installed and this is the last part left to enable it.

electronjoe commented:
Ekow can you fix the linting warnings?

electronjoe commented on .github/stale.yml:
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
```
Prefix entire file with --- per YAML lint warning

electronjoe commented on .github/stale.yml:
```diff
@@ -0,0 +1,20 @@
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - status: status: willfix
```
YAML linting doesn't like this line - are we able to quote `status: wilfix`? or do we need to disable YAML lint for this line in the file (done elsewhere in other workflow yaml files if you look around).

ekowtaylor replied:
I'll work around this

electronjoe commented on .github/stale.yml:
```diff
@@ -0,0 +1,20 @@
+staleLabel: status: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This pull request has been automatically marked as stale because it has not 
```
Here and elsewhere. YAML lint findings, drop trailing whitespace please!

ekowtaylor replied:
Will work around them all

ekowtaylor committed:
docs: Add stale.yml file for stalebot - Linting error fixes

Signed-off-by: Ekow Taylor <ekowtaylor@gmail.com>

This implements the config file (now with additional changes) for a GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.

ekowtaylor commented:
@electronjoe I just submitted a new commit which should address the issues
